modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -83,65 +124,155 @@ Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Ser
 |===
-|URL | Function | Requirement
+|URL | Function | Port
 
 |`mirror.openshift.com`
-|Required to access mirrored installation content and images. This site is also a source of release image signatures, although the Cluster Version Operator needs only a single functioning source.
-|Required
+|Used to access mirrored installation content and images. This site is also a source of release image signatures, although the Cluster Version Operator needs only a single functioning source. Required if you do not allow `storage.googleapis.com/openshift-release`.
+|443, 80
 
 |`storage.googleapis.com/openshift-release`
-|Alternative site to download platform release signatures that are used by the cluster to know what images to pull from quay.io.
-|Recommended
+|Recommended. Alternative site to mirror.openshift.com/. Used to download platform release signatures that are used by the cluster to know what images to pull from quay.io.
+|443, 80
+
+|`*.apps.<cluster_name>.<base_domain>`
+|Required. Used to access the default cluster routes unless you set an ingress wildcard during installation.
+|443, 80
 
 |`quay-registry.s3.amazonaws.com`
-|Used to access Quay image content in AWS.
-|Required
+|Required. Used to access Quay image content in AWS.
+|443, 80
 
 |`api.openshift.com`
-|Required to check if updates are available for the cluster.
-|Required
+|Required. Used to check if updates are available for the cluster.
+|443, 80
 
 |`art-rhcos-ci.s3.amazonaws.com`
-|Specifies the {op-system-first} images to download.
-|Required
+|Required. Specifies the {op-system-first} images to download.
+|443, 80
 
 |`cloud.redhat.com/openshift`
-|Required for your cluster token.
-|Required
+|Required. Used for cluster tokens.
+|443, 80
 
 |`registry.access.redhat.com`
-| Access to the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
-|Required
+|Required. Used to access the `odo` CLI tool that helps developers build on OpenShift and Kubernetes.
+|443, 80
+
+|`quayio-production-s3.s3.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`cm-quay-production-s3.s3.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`ec2.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`events.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`iam.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`route53.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`sts.amazonaws.com`
+|Required. Used to install and manage clusters in an AWS environment.
+|443, 80
+
+|`ec2.us-east-1.amazonaws.com`
+|Required. Region dependent and must be added per cluster and per region.
+|443, 80
+
+|`CLUSTER-NAME-k5bxz-image-registry-us-east-1-lsiflffxtmfyikx.s3.dualstack.us-east-1.amazonaws.com`
```

Comment:
```suggestion
|`CLUSTER-NAME-k5bxz-image-registry-us-east-1-lsiflffxtmfyikx.s3.dualstack.us-east-1.amazonaws.com`
```

This is also created at install time, customer will need to retrieve their endpoint via:

```
oc -n openshift-image-registry get pod -l docker-registry=default -o json | jq '.items[].spec.containers[].env[] | select(.name=="REGISTRY_STORAGE_S3_BUCKET")'
```

Reply:
I don;t see a difference in the lines, but assume you want me to replace `us-east-1` with <aws_region>, right?

Reply:
This endpoint is generated dynamically during installation like the shard etc. We should call this out to customer and advise how to retrieve it :)

```
[~ {stg} (dofinn:default)]$ oc -n openshift-image-registry get pod -l docker-registry=default -o json | jq '.items[].spec.containers[].env[] | select(.name=="REGISTRY_STORAGE_S3_BUCKET")'

{
  "name": "REGISTRY_STORAGE_S3_BUCKET",
  "value": "dofinn-tm5zn-image-registry-us-west-2-lxomiuyebfutxsfl"
}
{
  "name": "REGISTRY_STORAGE_S3_BUCKET",
  "value": "dofinn-tm5zn-image-registry-us-west-2-lxomiuyebfutxsfl"
}
```